src/plugins/dashboard/public/dashboard_container/component/grid/_dashboard_panel.scss

```
@@ -4,6 +4,10 @@
   * .embPanel--editing doesn't get updating without a hard refresh
   */
 
+.dshDashboardGrid__item {
+  scroll-margin-top: calc(var(--euiFixedHeadersOffset, 100)* 2);
+}
```

What do you think about adding `scroll-margin-bottom` for editing panels at the bottom of the dashboard? 🤔

```suggestion
  scroll-margin-bottom: $euiSizeS;
}
```